src/adv.py

```diff
@@ -49,3 +53,34 @@
 # Print an error message if the movement isn't allowed.
 #
 # If the user enters "q", quit the game.
+print(textwrap.dedent(
+    f"""\
+    You are in room: {player.current_room.name}
+    {player.current_room.description}!\
+    """))
+
+print(textwrap.dedent('''
+        What would you like to do?
+        [n] Go North [s] Go South [e] Go East [w] Go West
+        [take out <item>] or [return <item>]
+        [drop <item>] Remove item from bag
+        [b] look in bag
+        [q] Quit
+    '''))
+
+while True:
+    choice = input()
```

Comment:
You could print a message or add a message in the input parentheses to ask the player what they want to do next.